src/LogicApp/LogicApp.Management.Sdk/function-compare.ps1

```diff
@@ -0,0 +1,159 @@
+param(
```

Comment:
please remove this file

Reply:
Thank you for your help, it has now been removed.